keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/SecurityGroup.kt

```diff
@@ -2,13 +2,16 @@ package com.netflix.spinnaker.keel.api.ec2
 
 import com.netflix.spinnaker.keel.api.ExcludedFromDiff
 import com.netflix.spinnaker.keel.api.Moniker
+import com.netflix.spinnaker.keel.api.support.Tag
 
 data class SecurityGroup(
   val moniker: Moniker,
   val location: Location,
   @get:ExcludedFromDiff
-  val description: String?,
-  val inboundRules: Set<SecurityGroupRule> = emptySet()
+  val description: String? = moniker.toString(),
+  val inboundRules: Set<SecurityGroupRule> = emptySet(),
+  @get:ExcludedFromDiff
```

Comment:
My rationale was that I don't expect users to specify tags in the delivery config, but I needed them because that's the second place where we store the Object IP IDs.